addons/survey/wizard/survey_invite.py

```diff
@@ -165,6 +165,11 @@ def _compute_body(self):
                 invite = invite.with_context(lang=langs.pop())
             super(SurveyInvite, invite)._compute_body()
 
+    @api.depends('template_id.attachment_ids')
```

Comment:
Don't need "attachment_ids": we don't want to recompute all existing wizards in db if a template changes. Just "template_id" so that it is an onchange-like behavior.